chrome_extension/cardboard/popup.js

```diff
@@ -1,10 +1,37 @@
+const IS_PROD = true;
+
+const PROD_URL = "https://cardboard.rocks";
+const DEV_URL = "http://localhost:8000";
+
+// TODO: Make this configurable
+const HUNT_NUMBER = 10;
+
+let URL = DEV_URL;
+if (IS_PROD) {
+  URL = PROD_URL;
+}
+
+// Read all puzzles and filter down to those that are metas.
```

Comment:
I feel like we should probably change the API to allow directly querying for metas, maybe a TODO?

Reply:
This would be a TODO on the API side. I added https://github.com/cardinalitypuzzles/cardboard/issues/745